### PR TITLE
commit 2300 bug plus another

### PR DESCRIPTION
**Pull request by Goven112**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by tvardero — APPROVED** [review]

**Review by bkondrat — APPROVED** [review]

File: src/pages/Club/CreateClub/CreateClub.tsx [hunk 1]
```diff
@@ -156,6 +157,25 @@ const CreateClub = () => {
       });
   };
 
+  const sureConfirm = () => {
+    Modal.confirm({
+      title: "Ваші дані будуть не збережені.",
+      content: (
+        <div className={classes.Style}>
+      
+          <b>
```
Comment (Omeliukh-Nazar) [inline src/pages/Club/CreateClub/CreateClub.tsx:166]: Please, format this code

File: src/pages/Club/Club/Club.tsx [hunk 2]
```diff
@@ -501,6 +502,52 @@ const Club = () => {
       onOk() {},
     });
   };
+  const showConfirmAddNewHead = (newAdmin: ClubAdmin, existingAdmin?: ClubAdmin) => {
+    Modal.confirm({
+      title: "Призначити даного користувача на цю посаду?",
+      content: (
+        <div className={classes.Style}>
+            <b>
```
Comment (Omeliukh-Nazar) [inline src/pages/Club/Club/Club.tsx:510]: Format this

File: src/pages/City/CreateCity/CreateCity.tsx [hunk 3]
```diff
@@ -384,6 +385,25 @@ const CreateCity = () => {
     });
   }
 
+  const sureConfirm = () => {
+    Modal.confirm({
+      title: "Ваші дані будуть не збережені.",
+      content: (
+        <div className={classes.Style}>
+
```
Comment (Omeliukh-Nazar) [inline src/pages/City/CreateCity/CreateCity.tsx:393]: format this

Comment (Goven112) [reply src/pages/City/CreateCity/CreateCity.tsx]: done

**Review by Omeliukh-Nazar — APPROVED** [review]

**Pull request merged**